ddocs/medic/_attachments/translations/messages-en.properties

```
@@ -948,6 +948,7 @@ permission.description.can_view_tasks_tab = Display the Tasks tab in the app. If
 permission.description.can_view_unallocated_data_records = Allowed to see reports that have no assigned contact.
 permission.description.can_view_users = Allowed to get a list of all configured users.
 permission.description.can_write_wealth_quintiles = Allowed to update contacts with their wealth quintiles.
+permission.description.can_upgrade = Allowed to upgrade the application.
```

I don't think this message is clear enough, specifically, app developers build and upgrade the application using cht-conf. This permission is only for updating cht-core. Copying what you wrote for the docs PR is good.

```suggestion
permission.description.can_upgrade = Allowed to upgrade the CHT Core Framework version via the API or admin interface
```